packages/components/src/components/trace/TraceEventBlock.vue

```diff
@@ -102,6 +102,7 @@ export default {
 
       // Cache the expanded state on the event
       if (this.cacheState) {
+        // eslint-disable-next-line vue/no-mutating-props
         this.event.$hidden.expanded = this.expanded;
```

Comment:
Is `event.$hidden.expanded` even used anywhere? It'd be faster to do this, but I don't think we're actually using it.

Reply:
You're right, I removed it.